codecarbon/emissions_tracker.py

```diff
@@ -164,8 +165,9 @@ def _measure_power(self) -> None:
         """
         self._total_energy += Energy.from_power_and_time(
             power=self._hardware.total_power,
-            time=Time.from_seconds(self._measure_power_secs),
+            time=Time.from_seconds(time.time()-self._last_measured_time),
```

Comment:
Mypy is complaining that `self._last_measured_time` could be None as its type hints is `Optional[float]`. Even if it's unlikely that `_measure_power` would be called before `start` is called, if you want to play the safe card, check out the code suggestion here: https://github.com/mlco2/code-carbon/pull/34/files#diff-99ecda6f3fccf943e43b5939305579d7cef3734928186e5395b14c0eda681307R52